silo/views.py

```diff
@@ -439,16 +440,22 @@ def edit_silo(request, id):
     """
 
     edited_silo = Silo.objects.get(pk=id)
-
+    user_wfl1s = get_workflowlevel1s(request.user)
     request_user_org = None
     owner_user_org = None
+
     if(hasattr(request.user, 'tola_user') and
             hasattr(edited_silo.owner, 'tola_user')):
         request_user_org = request.user.tola_user.organization
         owner_user_org = edited_silo.owner.tola_user.organization
 
-    is_silo_shared_with_user = Silo.objects.filter(
-        pk=id, shared__id=request.user.pk).exists()
+    if user_wfl1s:
```

Comment:
I don't think you need this conditional. If the `user_wfl1s` is empty, only the `shared__id` will already be used in the filter.